packages/fakta-omsorgen-for/src/ui/components/omsorgsperiode-vurderingsdetaljer/OmsorgsperiodeVurderingsdetaljer.tsx

```diff
@@ -42,7 +43,12 @@ const OmsorgsperiodeVurderingsdetaljer = ({
           ? 'Søker er folkeregistrert forelder'
           : 'Søker er ikke folkeregistrert forelder';
       }
-      label = <Label size="small">Automatisk vurdert</Label>;
+      label = (
+        <div className="flex items-center gap-2">
```

Comment:
Stemning for å bruke Aksel fremfor tailwind der vi kan? HStack i dette tilfellet